src/components/MovieCard/MovieCard.tsx

```diff
@@ -2,6 +2,47 @@ import React from 'react';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  title: string;
+  description: string;
+  imgUrl: string;
+  imdbUrl: string;
```

Comment:
And you can pass whole `movie` as a one prop and use your `Movie` type, so you don't repeat yourself here